### DISCUSSION TITLE
Log name save

### DISCUSSION THREAD

**Discussion by rodrigompais** [spatie/laravel-activitylog#749] [Q&A] [answered]
Good afternoon people! How can I put the user name logged in to default_log_name?

**Comment by FallenKnight85** [comment]
Hello, why do you want to put the user logged in to default_log_name?

**Comment by Gummibeer** [comment, marked as answer]
We have several issue/discussion templates you can/have to choose one of. This one is by far not a bug report. Primary because you've removed the bug report template and only provided a single question without details.
So this is/would be a request for help discussion. Next time I will simply close wrongly opened bug reports.

------

You will have to override the default log name in the `tapActivity()` method for model logs and can pass the log name for custom logs into the `log()` method.
Or if you want to do this for every activitylog you can use an observer for the activity model.
